### PR TITLE
Sprint project

### PR DESCRIPTION
**Pull request by amymhaddad**
https://github.com/amymhaddad/Sprint-Challenge-Applied-Javascript

### CODE REVIEW

File: components/Cards/index.js [hunk 1]
```diff
@@ -17,3 +16,128 @@
...
+       }
+    }
+}
+
+
+function getArticleData(allArticleData) {
+    getArticleTopics(allArticleData)
+    
+}
+
```
Comment (paulghaddad) [inline components/Cards/index.js:117]: wrap in a function and then call similar to header and then call

File: components/Cards/index.js [hunk 2]
```diff
@@ -17,3 +16,128 @@
...
+axios.get("https://lambda-times-backend.herokuapp.com/articles")
+    .then(function(response) {
+
+        let allData = response.data['articles']
+        getArticleData(allData)
```
Comment (paulghaddad) [inline components/Cards/index.js]: getArticleDataAndCreateCards

**Comment by amymhaddad** [comment]
@paulghaddad All changes have been made. Pls see files for updates.

I have one question in the Cards/index.js file. Pls see my comment in the file itself.

File: components/Cards/index.js [hunk 3]
```diff
@@ -17,3 +16,113 @@
...
+
+const articleData = "https://lambda-times-backend.herokuapp.com/articles"
+
+const tabs = document.querySelector(".tabs");
+
+tabs.addEventListener("click", function(event) {
+    let tabName = event.target.innerHTML;
+    let allCards = document.querySelectorAll(".card")
+    
+    const hiddenArticles = document.querySelectorAll(".hide-articles")
```
Comment (amymhaddad) [inline components/Cards/index.js:30]: Not clear when ".hide-articles" gets added and how it knows which articles to add this class to?

**Pull request merged**